### PR TITLE
Jacob's Codingbat Exercises 1

### PR DESCRIPTION
**Pull request by ghost**
http://jacobthill.github.io/spring2014/

### CODE REVIEW

**Comment by gerbal** [comment]
@jacobthill you should only use the codeblocks for code. Because you've put all of your markdown in code-blocks Jekyll is ignoring the formatting and showing us the raw markdown.

File: _posts/2014-02-09-jacob-cobdingbat.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+author: jacobthill
+title: Coding Bat Exercises 1
+---
+
+```
+There weren't any specific exercises that I really liked above the others. I enjoyed all of them, but 
+I liked the Strings and Logic sections most. I think the Lists section was the hardest for me 
+and also the most useful. I was clear that I needed to review this material more than some of the other
+material we've covered. 
+```
```
Comment (eah13) [inline _posts/2014-02-09-jacob-cobdingbat.md]: Lines 7 and 12 are what grant's referring to.  If you leave out these backticks, the text will render normally and you wont ahve to press enter after every line.

**Comment by eah13** [comment]
@jacobthill Please post the code from 3 specific problems and your reflections on them.  See some of the other posts that have been merged if you're unclear on what that might mean: http://silshack.github.io/spring2014/

**Comment by eah13** [comment]
@jacobthill your three examples don't include reflections, and "There weren't any specific exercises that I really liked above the others" doesn't count.

**Pull request merged**